core/templates/components/state-editor/state-responses-editor/state-responses.component.html

```diff
@@ -167,7 +167,7 @@ <h3>Misconceptions</h3>
           <small class="text-muted">The following must be addressed by tagging to an incorrect answer. If a misconception is not relevant to this question, mark it as "Not Applicable".</small>
           <div class="pt-4" *ngIf="getUnaddressedMisconceptionNames(misconceptionsBySkill).length">
             <h4 class="pl-3">Compulsory</h4>
-            <p *ngFor="let misconceptionName of getUnaddressedMisconceptionNames(misconceptionsBySkill); let index = index" class="misconception-list-item">
+            <p *ngFor="let misconceptionName of getUnaddressedMisconceptionNames(misconceptionsBySkill)">
```

Comment:
Hi @StephenYu2018, is the styling change here expected i.e. removal of the class name?

Reply:
@kevintab95 Ah, my bad. I think I was trying to remove the `index` since it wasn't being used, and I accidentally removed the class name as well.